### PR TITLE
adding missing tests

### PR DESCRIPTION
**Pull request by stefandeml**

### CODE REVIEW

File: storage-proofs-zexe/src/circuit/test/mod.rs [hunk 1]
```diff
@@ -193,13 +193,13 @@ impl<E: Engine> TestConstraintSystem<E> {
...
         let negone = {
-            let mut tmp = E::Fr::one();
+            let tmp = E::Fr::one();
             // tmp.negate();
             -tmp
         };
 
         let powers_of_two = (0..  <E::Fr as PrimeField>::Params::MODULUS_BITS)
-            .map(|i| (E::Fr::from_repr_raw(<E::Fr as PrimeField>::BigInt::from(2 as u64))))
+            .map(|_| (E::Fr::from_repr_raw(<E::Fr as PrimeField>::BigInt::from(2 as u64))))
```
Comment (sanchopansa) [inline storage-proofs-zexe/src/circuit/test/mod.rs]: Isn't this always equal to two?

**Review by sanchopansa — COMMENTED** [review]
LGTM. I'm just wondering about the `power_of_two` construction.

File: storage-proofs-zexe/src/circuit/test/mod.rs (hunk 1 above)
Comment (stefandeml) [reply storage-proofs-zexe/src/circuit/test/mod.rs]: good catch, I think we missed this during our call. Fixed it!

**Comment by stefandeml** [comment]
Fixed the `power_of_two`construction

**Pull request merged**